build.sbt

```diff
@@ -34,11 +34,13 @@ inThisBuild(
   Seq(
     homepage := Some(url("https://github.com/gemini-hlsw/explore")),
     addCompilerPlugin(
-      ("org.typelevel" % "kind-projector" % kindProjectorVersion).cross(CrossVersion.full)
+      ("org.typelevel"                    % "kind-projector" % kindProjectorVersion).cross(CrossVersion.full)
     ),
     description := "Explore",
     scalacOptions += "-Ymacro-annotations",
-    Global / onChangedBuildSource := ReloadOnSourceChanges
+    Global / onChangedBuildSource := ReloadOnSourceChanges,
+    scalafixDependencies += "edu.gemini" %% "clue-generator" % Settings.LibraryVersions.clue,
+    scalafixScalaBinaryVersion := "2.13"
```

Comment:
Will this change in dotty?

Reply:
Yes, that's the scala version Scalafix runs in, which may or may not be the the same as the project on which it's being run. It defaults to 2.12 so we have to set it if it takes another value. In our case, we are only publishing the `GraphQLGen` rule for 2.13, so we have to change it.